### PR TITLE
A bunch of Pokemon sprite changes and fixes

### PR DESCRIPTION
**Pull request by cafei-uh**
## Description
- New Basculin Blue and White entry animations based on Red Stripe animation
- New Xerneas Neutral entry animation based on its Active form animation
- Recolored Iron Hands and Iron Bundle back sprites
- Fixed floating back sprite of Munkidori

## Images
TGAGcyfOazc 1  image
okngwzUUy18 1  image
P1k5qUe-UTk 1 image

https://github.com/rh-hideout/pokeemerald-expansion/assets/46283144/b0ac205d-b6f9-472b-9b67-f68e937408b5

https://github.com/rh-hideout/pokeemerald-expansion/assets/46283144/432f056c-8643-4f33-8ecc-10d7955b1f18

### CODE REVIEW

File: graphics/pokemon/iron_bundle/back.png
Comment (AsparagusEduardo) [inline graphics/pokemon/iron_bundle/back.png:1]: Our standard is to have the shiny palettes for thr backsprite pngs. Can you do that for this?

File: graphics/pokemon/iron_hands/back.png
Comment (AsparagusEduardo) [inline graphics/pokemon/iron_hands/back.png:1]: Here as well.

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]
Looks good! Just two things to tweak :)

File: graphics/pokemon/iron_hands/back.png
Comment (cafei-uh) [reply graphics/pokemon/iron_hands/back.png:1]: Done

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**